Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -586,45 +583,9 @@ internal struct LongPathInfo
             try
             {
                 this._isCheckingForLongPaths = true;
-                TaskDialogButton dedupeButton, ignoreButton, disableButton;
-                var taskDialog = new TaskDialog(NodejsPackage.Instance)
-                {
-                    AllowCancellation = true,
-                    EnableHyperlinks = true,
-                    Title = Resources.LongPathWarningTitle,
-                    MainIcon = TaskDialogIcon.Warning,
-                    Content = Resources.LongPathWarningText,
-                    CollapsedControlText = Resources.LongPathShowPathsExceedingTheLimit,
-                    ExpandedControlText = Resources.LongPathHidePathsExceedingTheLimit,
-                    Buttons = {
-                        (dedupeButton = new TaskDialogButton(Resources.LongPathNpmDedupe, Resources.LongPathNpmDedupeDetail)),
-                        (ignoreButton = new TaskDialogButton(Resources.LongPathDoNothingButWarnNextTime)),
-                        (disableButton = new TaskDialogButton(Resources.LongPathDoNothingAndDoNotWarnAgain, Resources.LongPathDoNothingAndDoNotWarnAgainDetail))
-                    },
-                    FooterIcon = TaskDialogIcon.Information,
-                    Footer = Resources.LongPathFooter
-                };
-
-                taskDialog.HyperlinkClicked += (sender, e) =>
-                {
-                    switch (e.Url)
-                    {
-                        case "#msdn":
-                            Process.Start("https://go.microsoft.com/fwlink/?LinkId=454508");
-                            break;
-                        case "#uservoice":
-                            Process.Start("https://go.microsoft.com/fwlink/?LinkID=456509");
-                            break;
-                        case "#help":
-                            Process.Start("https://go.microsoft.com/fwlink/?LinkId=456511");
-                            break;
-                        default:
-                            System.Windows.Clipboard.SetText(e.Url);
-                            break;
-                    }
-                };
 
-                recheck:
+                Utilities.ShowMessageBox(
```

Comment:
I might be missing something, but it looks like we show the warning *before* doing the long path check.